lib/logstash/codecs/compress_spooler.rb

```diff
@@ -6,12 +6,18 @@ class LogStash::Codecs::CompressSpooler < LogStash::Codecs::Base
   config :spool_size, :validate => :number, :default => 50
   config :compress_level, :validate => :number, :default => 6
 
+  # The amount of time in seconds since last flush before a flush is forced,
+  # on the next event.
+  # Values smaller than 0 disables time based flushing.
+  config :min_flush_time, :validate => :number, :default => 0
```

Comment:
as I understand, this changed behaviour, can we make it ```-1``` as default? this will make the setting backwards compatible, right? what do you think?